test/nearleyc.test.js

```diff
@@ -60,15 +65,27 @@ describe("bin/nearleyc", function() {
     });
 
     it('builds for TypeScript', function() {
-        this.timeout(10000); // It takes a while to run tsc!
+        this.timeout(100000); // It takes a while to run tsc!
         const {outPath, stdout, stderr} = externalNearleyc("grammars/typescript-test.ne", ".ts");
         expect(stderr).toBe("");
         expect(stdout).toBe("");
-        sh(`tsc ${outPath}.ts`);
+        sh(`tsc ${outPath}.ts --noEmitOnError`);
+        expect(stderr).toBe("");
```

Comment:
I don't think this will do what you think it does: `stderr` is still the output from `nearleyc` :)